docs/latest/configuring/data-source.md

```diff
@@ -130,6 +130,31 @@ window.config = {
 };
 ```
 
+The configuration can also be written as a JS Function in case you need to inject dependencies like external services:
+
+```js
+window.config = ({ randomService } = {}) => {
```

Comment:
Wouldn't `randomService` actually be the `ServicesManager`?

Reply:
Bad example =S, updated. Thanks.